cadcdata/setup.py

```diff
@@ -31,53 +27,28 @@
 metadata = dict(conf.items('metadata'))
 
 PACKAGENAME = metadata.get('package_name', 'packagename')
-DESCRIPTION = metadata.get('description', 'Astropy affiliated package')
+DESCRIPTION = metadata.get('description', 'CADC package')
 AUTHOR = metadata.get('author', '')
 AUTHOR_EMAIL = metadata.get('author_email', '')
 LICENSE = metadata.get('license', 'unknown')
-URL = metadata.get('url', 'http://astropy.org')
+URL = metadata.get('url', 'http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca')
 
 # Get the long description from the package's docstring
 __import__(PACKAGENAME)
 package = sys.modules[PACKAGENAME]
 LONG_DESCRIPTION = package.__doc__
 
-# Store the package name in a built-in variable so it's easy
-# to get from other parts of the setup infrastructure
-builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
-
 # VERSION should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-VERSION = metadata.get('version', '1.0.dev')
-
-# Indicates if this version is a release version
-RELEASE = 'dev' not in VERSION
+VERSION = metadata.get('version', 'none')
 
-if not RELEASE:
-    VERSION += get_git_devstr(False)
-
-# Populate the dict of setup command overrides; this should be done before
-# invoking any other functionality from distutils since it can potentially
-# modify distutils' behavior.
-cmdclassd = register_commands(PACKAGENAME, VERSION, RELEASE)
-
-# Freeze build information in version.py
-generate_version_py(PACKAGENAME, VERSION, RELEASE,
-                    get_debug_option(PACKAGENAME))
+# generate the version file
+with open(os.path.join(PACKAGENAME, 'version.py'), 'w') as f:
+    f.write('version = \'{}\''.format(VERSION))	
```

Comment:
I would have written this as:

"""'{}'""".format(VERSION)

so not using the \ escape.